README.md

```diff
@@ -1,87 +1,105 @@
-# Buildkite Test Splitter
+# Buildkite Test Engine Client
 
-Buildkite Test Splitter is an open source tool to orchestrate your test suites. It uses your Buildkite Test Analytics suite data to intelligently partition and parallelise your tests. 
-Test Splitter currently only supports RSpec but support for other frameworks is coming soon.
+Buildkite Test Engine Client (bktec) is an open source tool to orchestrate your test suites. It uses your Buildkite Test Engine suite data to intelligently partition and parallelise your tests.
 
-## Migrating to 0.9.0
-Version 0.9.0 introduces a new feature that requires Test Splitter to read test results from the runner for retries and verification purposes. To enable this feature, it is necessary to configure the `BUILDKITE_SPLITTER_RESULT_PATH` environment variable. This variable specifies the location of where the runner should store test results.
+bktec supports RSpec and Jest.
 
-Furthermore, we have updated the default test command for RSpec to `bundle exec rspec --format progress --format json --out {{resultPath}} {{testExamples}}`. Test Splitter will automatically replace `{{resultPath}}` with the value specified in `BUILDKITE_SPLITTER_RESULT_PATH`. If you want to customize the RSpec command, make sure to include `--format json --out {{resultPath}}` in the command. 
+## Migrating to 1.0.0
+Version 0.9.0 introduces a new feature that requires bktec to read test results from the runner for retries and verification purposes. To enable this feature, it is necessary to configure the `BUILDKITE_TEST_ENGINE_RESULT_PATH` environment variable. This variable specifies the location of where the runner should store test results.
```

Comment:
Hmm now that I see the changes, I don't think this make sense. How about grouping the new manadatory env variables for the `1.0.0`?
```
Following environment variables is now required
- `BUILDKITE_TEST_ENGINE_RUNNER`
    {{description}}
- `BUILDKITE_TEST_ENGINE_RESULT_PATH`
   {{description}}

In addition, version 1.0.0 introduces name changes to environment variables....
| Old variable | New variable |
....

@wooly what do you think?

Reply:
This would be a bit nicer!

Reply:
Thanks for the feedback @ags I'll group the required variables for better readability. Also great to hear you upgraded to v1.0.0-rc3